### PR TITLE
Fix uploads with path separators not working correctly on Windows

### PR DESCRIPTION
**Pull request by kbolashev**
We were using Python's `os.path`, that has os-dependent separators, when we should have used `posixpath`, because the paths sent to the server are POSIX.

### CODE REVIEW

**Comment by kbolashev** [comment]
[DDA Windows issues while uploading nested folders](https://trello.com/c/Z78wqT7f/1544-dda-windows-issues-while-uploading-nested-folders)

File: dagshub/upload/wrapper.py [hunk 1]
```diff
@@ -427,7 +428,7 @@ def add_dir(self, local_path, glob_exclude=""):
         for root, dirs, files in os.walk(local_path):
```
Comment (idonov8) [inline dagshub/upload/wrapper.py:428]: Could you replace `os.walk` here with something with `posixpath`?

**Review by idonov8 — APPROVED** [review]
Looks good overall, I'd consider removing the `import os` entirely if possible

File: dagshub/upload/wrapper.py (hunk 1 above)
Comment (kbolashev) [reply dagshub/upload/wrapper.py:428]: Doesn't look like it's possible, but maybe I can cast the path to a posixpath

Comment (idonov8) [reply dagshub/upload/wrapper.py:428]: COOL, MERGE IT

Comment (kbolashev) [reply dagshub/upload/wrapper.py:428]: nvm I already return a posixpath at that point

**Pull request merged**